src/clojurebridge_turtle/core.clj

```diff
@@ -43,11 +43,12 @@
   ([name color]
      (let [n (keyword name)]
        (when-not (n @turtles)
-         (swap! lines assoc n [])
-         (swap! turtles assoc n {:x 0
-                                 :y 0
-                                 :angle 90
-                                 :color color}))
+         (dosync
+          (swap! lines assoc n [])
+          (swap! turtles assoc n {:x 0
+                                  :y 0
+                                  :angle 90
+                                  :color color})))
        {n (n @turtles)})))
```

Comment:
Nope, dosync is not needed here.  I was working from a file that predated brandonbloom's changes.  But when I generated the pull request, I saw he had taken that out, and so I added further commits to the pull request to mimic what Brandon did.  So the pull request includes one commit that appears to put the dosync back in, followed by one that takes it out -- no net change in the overall diff.